clr/main.py

```diff
@@ -11,15 +49,32 @@ def main(argv=None):
 
     namespace_key, cmd_name = resolve_command(query)
     namespace = get_namespace(namespace_key)
-    bound_args = namespace.parse_args(cmd_name, argv[2:])
-
-    # Call the command.
-    if hasattr(namespace.instance, "cmdinit"):
-        namespace.instance.cmdinit()
-    result = namespace.command_callables[cmd_name](
-        *bound_args.args, **bound_args.kwargs
-    )
-    # Support exit codes. This supports int and bool return values.
-    if isinstance(result, (int, bool)):
-        # print(f"Result: {result}", file=sys.stderr)
-        sys.exit(result)
+
+    # Default successful exit code.
+    exit_code = 0
+
+    with init_beeline(namespace_key, cmd_name):
```

Comment:
@mpcusack-color If this code is run without honeycomb in the env then it fails with `RuntimeError: generator didn't yield`